### PR TITLE
Docker File updates

### PR DESCRIPTION
**Pull request by njogz**
- Use an env variable to set `DATAEMON_INITAL_PACKAGE` instead of a hard-coded string
- Add some initialisation scripts for superset

### CODE REVIEW

File: superset/Dockerfile [hunk 1]
```diff
@@ -5,5 +5,15 @@ USER root
 RUN pip install psycopg2
 
 RUN pip install psycopg2-binary
+
+RUN superset superset fab create-admin \
+               --username admin \
+               --firstname Superset \
+               --lastname Admin \
+               --email admin@superset.com \
+               --password admin
```
Comment (garethbowen) [inline superset/Dockerfile:14]: We should probably force users to specify an admin password in their environment variables.

**Review by garethbowen — CHANGES_REQUESTED** [review]
Nice one! One suggestion...

**Comment by andrablaj** [comment]
@njogz who could help review this PR while Gareth is on vacation, to have things moving forward?

File: README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ COUCHDB_PASSWORD=<password>
 COUCHDB_DB=<main database with all records>
 COUCHDB_HOST=<hostname for couchdb server>
 COUCHDB_PORT=<couchdb port>
+CHT_PIPELINE_BRANCH_URL=<The CHT Pipeline  branch to use e.g. https://github.com/medic/cht-pipeline.git#first_release>
```
Comment (derickl) [inline README.md:23]: Nitpick: would it really be a branch url if we just use the default repo url which defaults to main?

File: superset/Dockerfile [hunk 3]
```diff
@@ -2,8 +2,21 @@ FROM apache/superset:2.0.0
 # Switching to root to install the required packages
 USER root
 
+ARG SUPERSET_PASSWORD
+ENV SUPERSET_PASSWORD $SUPERSET_PASSWORD
+
 RUN pip install psycopg2
 
 RUN pip install psycopg2-binary
```
Comment (derickl) [inline superset/Dockerfile:10]: I do not think we need to install the two psycopg2 variants. The former would be used when you need to build it from source while the latter is used when you want a pre-built package https://pypi.org/project/psycopg2-binary/

psycopg2-binary would be appropriate for this scenario.

**Review by derickl — APPROVED** [review]
Looks good.

I've left some suggestions inline

**Review by garethbowen — APPROVED** [review]
Nice work. Ship it.

**Pull request merged**